components/dropdown/dropdown-content-mixin.js

```diff
@@ -244,7 +276,20 @@ export const DropdownContentMixin = superclass => class extends RtlMixin(supercl
 	}
 
 	close() {
-		this.opened = false;
+		const hide = () => {
+			this._closing = false;
+			this._showBackdrop = false;
+			this.opened = false;
+		};
+
+		if (!reduceMotion && mediaQueryList.matches && this.mobileTray && isVisible(this)) {
+			this.shadowRoot.querySelector('.d2l-dropdown-content-width')
+				.addEventListener('animationend', hide, { once: true });
+			this._closing = true;
+			this._showBackdrop = false;
```

Comment:
`hide()` also sets `_showBackDrop` to `false` -- is it still needed here? I guess it depends whether you want the backdrop to go away before/while it's hiding.

Reply:
Yep, we actually need the backdrop,shown to be false, not just remove it from the DOM. The reasoning behind this has to do with the backdrop preventing background scroll when opened. On backdrop.shown = true, the scrolling is locked, and only shown = false can release it (not just deleting from the DOM). I spent a bunch of time debugging this as the screen seemed to 'lock' in place after using the dropdown, because you do manually need to set the backdrop to go away.

Reply:
We might be able to add a `disconnectedCallback()` to `d2l-backdrop` to do some cleanup for that scenario.  It get strange if the backdrop instance is removed from the DOM and then re-added however.  I think this was originally purpose built for dialog and then refactored, and this scenario was probably avoided in the interest if scope, and not wanting to deal with those scenarios.